esmvalcore/preprocessor/_multimodel.py

```diff
@@ -407,6 +418,9 @@ def multi_model_statistics(products, span, statistics, output_products=None):
         For internal use only. A dict with statistics names as keys and
         preprocessorfiles as values. If products are passed as input, the
         statistics cubes will be assigned to these output products.
+    keep_originals: bool
+        If True, the output_products will include the input products.
+        If False, only the statistics products will be returned.
```

Comment:
```suggestion
    keep_input_datasets: bool
        If True, the output will include the input datasets.
        If False, only the computed statistics will be returned.
```
It would be good to write this documentation with the diagnostic developer in mind, i.e. someone does not have an extremely in-depth understanding of how the ESMValCore preprocessor is run. The above is only a suggestion, feel free to change it, but make sure to keep in mind who the target audience of this documentation is.